CMakePresets.json

```diff
@@ -11,7 +11,8 @@
             "hidden": true,
             "environment": {
                 "CCACHE_BASEDIR": "${sourceDir}",
-                "CCACHE_SLOPPINESS": "pch_defines,time_macros"
+                "CCACHE_SLOPPINESS": "pch_defines,time_macros",
+                "CMAKE_EXPORT_COMPILE_COMMANDS": "YES"
```

Comment:
Please do not mix ccache enviroment variables with others.

also the export compile commands should be a cacheVariable if there at all.

Reply:
OK